### PR TITLE
v1.2.0

### PR DESCRIPTION
**Pull request by tsharp**

### CODE REVIEW

File: AutoNumber/Properties/AssemblyInfo.cs [hunk 1]
```diff
@@ -9,9 +9,9 @@
 [assembly: AssemblyTitle("AutoNumber")]
 [assembly: AssemblyDescription("")]
 [assembly: AssemblyConfiguration("")]
-[assembly: AssemblyCompany("")]
+[assembly: AssemblyCompany("Celedon Partners")]
 [assembly: AssemblyProduct("AutoNumber")]
-[assembly: AssemblyCopyright("Copyright ©  2015")]
+[assembly: AssemblyCopyright("Copyright © 2015")]
```
Comment (BrettRojas) [inline AutoNumber/Properties/AssemblyInfo.cs]: minor but can you update this to 2017

File: AutoNumber/CreateAutoNumber.cs [hunk 2]
```diff
@@ -39,81 +38,81 @@ public class CreateAutoNumber : CeledonPlugin
...
+			RegisterEvent(PipelineStage.PostOperation, PipelineMessage.Create, "cel_autonumber", Execute);
 		}
 
-		protected void Execute(LocalPluginContext Context)
+		protected void Execute(LocalPluginContext context)
 		{
-			Trace("Get Target record");
-			Entity Target = Context.GetInputParameters<CreateInputParameters>().Target;
-			string pluginName = String.Format(PLUGIN_NAME, Target.GetAttributeValue<string>("cel_entityname"));
+		    context.Trace("Get Target record");
```
Comment (BrettRojas) [inline AutoNumber/CreateAutoNumber.cs]: not an immediate request but might pull these trace calls

File: AutoNumber/Constants/PipelineStage.cs [hunk 3]
```diff
@@ -0,0 +1,32 @@
+﻿/*The MIT License (MIT)
+
+Copyright (c) 2015 Celedon Partners
```
Comment (BrettRojas) [inline AutoNumber/Constants/PipelineStage.cs]: 2017?

File: AutoNumber/Constants/PipelineMessage.cs [hunk 4]
```diff
@@ -0,0 +1,37 @@
+﻿/*The MIT License (MIT)
+
+Copyright (c) 2015 Celedon Partners
```
Comment (BrettRojas) [inline AutoNumber/Constants/PipelineMessage.cs]: 2017?

File: AutoNumber/AutoNumberPluginConfig.cs [hunk 5]
```diff
@@ -1,6 +1,26 @@
-﻿using System;
-using System.Linq;
-using System.Collections.Generic;
+﻿/*The MIT License (MIT)
+
+Copyright (c) 2015 Celedon Partners
```
Comment (BrettRojas) [inline AutoNumber/AutoNumberPluginConfig.cs]: 2017?

File: AutoNumber/CreateAutoNumber.cs (hunk 2 above)
Comment (tsharp) [reply AutoNumber/CreateAutoNumber.cs]: Will put this in the backlog :)

File: AutoNumber/AutoNumberPluginConfig.cs (hunk 5 above)
Comment (tsharp) [reply AutoNumber/AutoNumberPluginConfig.cs]: Finished

**Pull request closed without merging**